### DISCUSSION TITLE
What is the build directory in the content?

### DISCUSSION THREAD

**Discussion by exbucks** [microsoft/vcpkg#33771] [General]
cmake -B [build directory] -S . "-DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]/scripts/buildsystems/vcpkg.cmake"

**Comment by Neumann-A** [comment]
This is not even a vcpkg related question. Read the cmake docs:
https://cmake.org/cmake/help/latest/manual/cmake.1.html#generate-a-project-buildsystem
https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-B

The build dir is the dir cmake will generate.
